lib/components/src/controls/helpers.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Adds `control` prefix to make ID attribute more specific.
+ * Removes spaces because spaces are not allowed in ID attributes
+ * @link http://xahlee.info/js/html_allowed_chars_in_attribute.html
+ * @example getControlId('my prop name') -> 'control-my-prop-name'
+ */
+export const getControlId = (value: string) => `control-${value.replace(/\s+/g, '-')}`;
```

Comment:
Can you add some unit tests for this? I think you're missing some cases. Examples:

https://github.com/ComponentDriven/csf/blob/next/src/index.test.ts#L3